<!DOCTYPE html>
<html>

    <head>
        <!-- **************************************************************************** -->
        <!-- SITE SETTINGS -->
        <!-- **************************************************************************** -->

        <!-- Page title -->
        <title style="color:black">Dest Tutorial</title>

        <!-- Site metadata -->
        <meta charset='UTF-8'>

        <!-- **************************************************************************** -->
        <!-- STYLES -->
        <!-- **************************************************************************** -->
        <link rel="stylesheet" type="text/css" href="../../css/custom_QA.css" />
    </head>

    <body>
        <div class="classDiv_content">

			<h1 option style="color:black"><b>Dest Tutorial</b></h1><br>

			<div class="classDiv_step">
				<p class="classP_step"><b><big>Overview</b></big></p>
				<ol type="1">
					<a href="#1"><li>First of all: data selection</li></a>
					<a href="#2"><li>Setting the visualization mode</li></a>
					<a href="#3"><li>Example questions which can be answered using Dest</li></a>
					<ul style="list-style-type:none">
						<a href="#32"><li>3.1. Does my region of interest show adaptive functional differences between populations?</li></a>
						<a href="#33"><li>3.2. What information is available for my gene of interest?</li></a>
						<a href="#36"><li>3.3. Is there any polymorphic inversion overlapping my region of interest? If so, what is the frequency of the different genotypes among populations?</li></a>
					</ul>
					<a href="#4"><li>Downloading track data</li></a>
					<a href="#5"><li>Saving the current browser instance</li></a>
					<a href="#6"><li>Uploading custom tracks</li></a>
				</ol>
			</div>
			<br><br>

			<p class="classP_justified">Prior to start browsing the Dest database, it is highly recommended to read through the &ldquo;<a href="./html/custom_DataDescription.html" target="_blank">Data description</a>&rdquo; and &ldquo;<a href="./html/custom_TracksDescription.html" target="_blank">Tracks description</a>&rdquo; sections of the Help. <!-- For an exhaustive review on molecular population genetics see Casillas S. and Barbadilla A. (2017) Molecular Population Genetics. <i>Genetics: in press</i>. --><br><br>Dest stores +N tracks with information about N population<i>Drosophila melanogaster</i> populations. <br><br>Before attempting to answer any biological question, you need to decide for which population(s) data is going to be displayed. This can be done through the &ldquo;Data selection&rdquo; section from the left-side menu of the &ldquo;. Within this section, tracks can be filtered according to population. Keep in mind that filters are added accumulatively!
			<br>
			<br>
			<div id="1">
				<p><b><big>1. First of all: data selection</b></big></p>
			</div>
			For the sample questions proposed in this document, the X population is used.
			<b>1.</b> A&gt; 
			<b>2.</b> B &gt; 
			<b>3.</b> C (Figure 1).</p><br>
			IMAGE HERE	
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/1.png" width="60%" height="60%" style="min-width:60em;"></div> -->
			<p class="classP_footer"><b>Figure 1.</b> Data selection: X population.</p>
			<br>
			<br><br>

			<div id="2"><p><big><b>2. Setting the visualization mode</b></big></p></div>

			<p class="classP_justified">Once you have decided the population(s) of interest, it is time to choose the proper visualization mode. Most quantitative tracks have been computed at different resolution scales (ranging from 1 to 100 kb) (Figure 2). The appropriate window size selection depends on the size of the region of interest. In general, to inspect whole-genome patterns, it is recommended to use 100 kb or 50 kb windows, whereas if the analysis is focused on a small region, 10 kb or even 1 kb windows would be preferred.</p><br>
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/2.png" width="15%" height="15%" style="min-width:15em;"></div> -->
			IMAGE HERE
			<p class="classP_footer"><b>Figure 2.</b> Visualization mode (window size) selection.</p>
			<br>

			<div id="3"><p><big><b>3. Example questions which can be answered using Dest:</b></big></p><div id="1">
			
			THESE ARE THE SAME EXAMPLES AT POPFLY. IF YOU WANT TO SHOW ANOTHER REGION/GENE WITH ANY EVIDENCE OF SELECTION. LET ME KNOW.

			<p class="classP_justified">Dest is designed to help testing evolutionary hypotheses from a population genetics perspective. Below are some examples of questions that can be answered using Dest. Questions are sequential and the whole guide constitutes an example of how to analyze a specific region of the genome.<br><br>The region of interest is <b>2L:5022509..5047291</b>, a genomic area of around 25 kb placed in chromosome arm 2L, where the <i>Cg25C</i> gene is located. This region has been reported to be under positive selection (Garud <i>et al</i>., 2015; Garud and Petrov 2016). Moreover, <i>Cg25C</i> gene is a well-studied protein coding gene which belongs to the type IV collagen family, related to immune system activation (Kiss <i>et al</i>., 2016) and to cold adaptation in Drosophila (MacMillan <i>et al</i>., 2016).
			<br>
			<br>
			To display the target region, type the gene name at the coordiantes box. It is possible to highlight the region of interest using the corresponding utility located next to the genomic coordinates box. This allows exploring broader genomic areas without losing the region of interest.</p>
			<br>
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/3.png" width="80%" height="50%" style="min-width:50em;"></div>
			<p class="classP_footer"><b>Figure 3.</b> Genomic coordinates box and highlight utility button</p> -->
			<br>
			<br>
			<div id="32"><p class="classP_justified2"><b>3.1. Does my region of interest show adaptive functional differences between populations?</b></p></div>
			<p class="classP_justified2">Usually, selective sweeps are due to adaptive variants that have been selected in a specific population. As a result, this specific population becomes genetically differentiated in the selected region (high 
				<span class="tooltip_link">fst
					<span class="classic_tt">
						F<sub>ST</sub>
					</span>
				</span>
				 values). To test if the region of interest shows adaptive functional differences between populations, 
				<span class="tooltip_link">fst
					<span class="classic_tt">
						F<sub>ST</sub> estimates
					</span>
				</span>
				 estimates (located in the &ldquo;Population differentiation&rdquo; sub-section of the &ldquo;Track selector&rdquo; tab) should be considered.<br><br>We examine 
				<span class="tooltip_link">fst
					<span class="classic_tt">
						F<sub>ST</sub>
					</span>
				</span>
				 values between A and B, C, D and E populations. Since A and B are cosmopolitan populations it is expected to observe higher population differentiation values between A and ancestral ones (C, D and E.</p>
			<br>
			<div class="classDiv_step">
			IMAGE HERE
			<div id="33"><p class="classP_justified2"><b>3.2. What information is available for my gene of interest?</b></p></div>
			<p class="classP_justified2">The Dest browser includes a report for each gene annotation, as well as direct links to the databases NCBI and FlyBase. Hence, information regarding location, structure, sequence, gene ontology, gene expression, literature about the gene of interest, etc. can be accessed directly from the browser interface.<br><br>By right-clicking the gene of interest, a drop-down menu appears with several options (including the ones to search for the specific feature at both NCBI and FlyBase databases). These options can also be accessed from the &ldquo;View details&rdquo; dialog (Figure 7). Moreover, note that it is possible to search for a specific feature name (such as a gene symbol) using the genomic coordinates box and pressing the &ldquo;Go&rdquo; button.</p><br>
			IMAGE HERE
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/9.1.png" width="50%" height="50%" style="min-width:50em;"></div>
			<div class="classDiv_img"><img src="../../img/custom_QA/9.2.png" width="50%" height="30%" style="min-width:30em;"></div>
			<p class="classP_footer"><b>Figure 7.</b> Feature search box (top) and &ldquo;View details&rdquo; dialog box (bottom) with links to NCBI and FlyBase databases.</p> -->
			<br>
			<div id="36"><p class="classP_justified2"><b>3.3. Is there any polymorphic inversion overlapping my region of interest? If so, what is the frequency of the different genotypes among populations?</b></p></div>
			<p class="classP_justified2">Dest includes a track with annotations of polymorphic 
				<span class="tooltip_link">inversions
					<span class="classic_tt">
						Coordinates of the 19 most studied polymorphic inversions in the <i>Drosophila melanogaster</i> genome (Corbett-Detig and Hartl 2012; Huang <i>et al</i>., 2014).
					</span>
				</span>
				 identified in the <i>D. melanogaster</i> genome (Corbett-Detig and Hartl 2012; Huang <i>et al</i>., 2014). This track is located in the &ldquo;Reference tracks&rdquo; section of the &ldquo;Track selector&rdquo; tab.<br><br>As it is known that most inverted haplotypes in <i>Drosophila</i> have emerged much recently than the standard ones (assuming that the inverted haplotype is the derived allele), it may be critical to inspect the frequency distribution of such haplotypes in any given population when identifying, for example, a region of low diversity (putative selective sweep), since the signature of decreased nucleotide diversity levels may instead be due to the low recombination as a result of an inversion (Corbett-Detig and Hartl 2012; Huang <i>et al</i>., 2014).<br><br>Note that one inversion overlaps the region of interest. Right-click the inversion annotation to visualize all the options in a drop-down menu. Information about this inversion can be visualized in the options &ldquo;View details&rdquo; and &ldquo;View frequency distribution among populations&rdquo; (Figure 10).</p><br>
			<br>
			IMAGE HERE
			<br>
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/8.1.png" width="45%" height="45%" style="min-width:50em;"></div>
			<div class="classDiv_img"><img src="../../img/custom_QA/8.2.png" width="40%" height="40%" style="min-width:45em;"></div>
			<p class="classP_footer"><b>Figure 10.</b> Inversion genotypes frequency distribution among populations (top: accessing the inversion dialog; bottom: inversion&rsquo;s frequency dialog box).</p> -->
			<br>
			<div id="4"><p><b><big>4. Downloading data</b></big></p></div>
			<p class="classP_justified">To download track data or pool-seq data from the browser interface, place the mouse over the track title (on the left), press the arrow which appears to the right of the track label, and select the &ldquo;Save track data&rdquo; option. In the dialog, define the region of interest (visible region or whole reference sequence), the output format (bedGraph, wiggle or gff3) and file name, and press the &ldquo;Save&rdquo; button (Figure 11).</p><br>
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/11.png" width="80%" height="30%" style="min-width:15em;"></div>
			<p class="classP_footer"><b>Figure 11.</b> Downloading track data.</p><br> -->
			<br><br>
			<p>Dest incorporates an utility specifically designed for easy and fast downloading of pool-seq data or tracks information. This utility can be accessed from the main browser interface by clicking at <i>Tools</i> options.
			The utility redirects the user to a specific web-page create to easy download the coordinates which are being displayed  at the moment you click or another any coordinates. It is required to provide the genomic coordinates (chr, start, and end) of the target region, as well as the population of interest.
			<br>
			IMAGE HERE
			<br>

			<div id="5"><p><b><big>5. Saving the current browser instance</b></big></p></div>
			<p class="classP_justified">Dest allows saving any particular browser instance to be shared or visualized later on. This can be done through the &ldquo;Share&rdquo; button located in the top-right corner of the browser interface (Figure 12). When clicking it, the user can copy the corresponding URL, which will render the same exact Dest instance that is being displayed now.</p><br>
			IMAGE HERE
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/12.png" width="80%" height="30%" style="min-width:15em;"></div>
			<p class="classP_footer"><b>Figure 12.</b> Saving the current browser instance.</p><br> -->
			<br><br>
			<p><b><big>Dest share URL with all activated tracks used in this guide:</b> <a href="//Dest.uab.cat/?loc=2L%3A4983801..5067200&tracks=gene_annotations%2CZI_Pi_1kb%2CZI_Theta_1kb%2CZI_Kdyak_1kb%2CZI_r2_1kb%2CZI_TajD_1kb%2CZI_EA_fst_1kb%2CZI_FR_fst_1kb%2CZI_RAL_fst_1kb%2CZI_SP_fst_1kb%2CZI_DoS_10kb%2CZI_FisherPval_10kb%2CZI_KaKs_10kb%2CZI_MKalpha_10kb%2CZI_NI_10kb%2Cinversions&highlight=2L%3A5028900..5038399" target="_blank">link</a></big></p> 
			<br><br>
			
 			<div id="6"><p><b><big>6. Uploading custom tracks</b></big></p></div>
			<p class="classP_justified"></p>To upload you own tracks, press the &ldquo;Track&rdquo; button from the navigation menu located in the top-left area of the browser interface. Then, select the &ldquo;Open track file or URL&rdquo; option. Next, browse your local file or paste a remote URL in the corresponding box and customize the file type and display options. Finally, select whether to open immediately your custom track or add it to the tracks set and press the &ldquo;Open&rdquo; button.</p><br>			
			<!-- <div class="classDiv_img"><img src="../../img/custom_QA/13.png" width="80%" height="30%" style="min-width:15em;"></div>
			<div class="classDiv_img"><img src="../../img/custom_QA/14.png" width="80%" height="30%" style="min-width:15em;"></div>
			<p class="classP_footer"><b>Figure 13.</b> Uploading custom tracks.</p><br> -->
			<br><br>
		</div>
    </body>
</html>