<!DOCTYPE html>
<html>

    <head>

        <!-- **************************************************************************** -->
        <!-- SITE SETTINGS -->
        <!-- **************************************************************************** -->

        <!-- Page title -->
    	<title>Data Description</title>

        <!-- Site metadata -->
    	<meta charset='UTF-8'>

    	<!-- **************************************************************************** -->
        <!-- STYLES -->
        <!-- **************************************************************************** -->
        <link rel="stylesheet" type="text/css" href="../../css/custom_main.css" />
        <style>body {max-width: 1000px;}</style>

    </head>

    <body>

        <div class="classDiv_help_dialog">
            <div class="mainData">
            <dl>
                <p align="justify">We assembled 246 Pool-Seq libraries of population samples from Europe, North America and the Caribbean which were densely sampled through time and space by two collaborative population genomics consortia in North America (DrosRTEC: https://web.sas.upenn.edu/paul-schmidt-lab/dros-rtec/) and Europe (DrosEU: http://droseu.net) between 2003 and 2016. In addition, we integrated genomic data from >900 inbred or haploid genomes from African, European, Australian, and North American available from the Drosophila Genomes Nexus  (DGN; Lack et al. 2015, 2016), making this repository the largest concatenated dataset of whole-genome SNP polymorphisms available for D. melanogaster to date. 
                </p>

                <p align="justify">Two complementary strategies have been implemented to maximize the number of trustworthy low-frequency variants while minimizing the number of false-positive sequencing errors. Accordingly, heuristic approaches such as VarScan (cite), PoPoolation (cite) or Pool-SNP (cite) integrate the allelic information across multiple independent samples and apply stringent global allele count and allele frequency thresholds for variant detection. This approach allows to detect common variants with a low overall frequency above a given threshold, but is not very sensitive to rare private alleles. Conversely, probabilistic approaches, as implemented in SNAPE (cite), use a priori information about genetic variation in a given population as a prior to distinguish sequencing errors from real SNPs using Bayesian statistics. Complementary to the heuristic approaches described above, SNAPE calls SNP variants in single samples which facilitates the detection of private alleles but does not allow to take into account information from other samples. In our automated analysis pipeline, we take advantage of both approaches and implement the methods of Pool-SNP and SNAPE in our pipeline and compare their performance.
                Specifically, we consider a position polymorphic if (1) the cumulative allele count of a minor allele across all samples is ≥ 50, if (2) the cumulative allele frequency is ≥ 0.001 and (3) less than 50% of all samples have missing data at the given position. SNP polymorphisms are then annotated with SNPeff (v. XXX; Cingolani et al. 2012) and stored in VCF and BCF (Danecek et al. 2011) file formats alongside an index file in TABIX format (Li 2011). VCF files were further converted to GDS format [cite] using the R package SeqArray [cite].
                </p>

                <p align="justify">The N analyzed 1000GP populations are listed below:</p>
 
                <p align="center">
                  <table style="width:100%">
                  </table>

                    SAMPLES IMAGE
<!-- <br><img src="http://www.internationalgenome.org/sites/1000genomes.org/files/images/1000g_map.png" style="width:100%"><br> -->

            </dl>
            </div>
        </div>

    </body>

</html>




    
